tabpfn_client/prompt_agent.py

```diff
@@ -74,22 +84,23 @@ def prompt_terms_and_cond(cls) -> bool:
             "By using TabPFN, you agree to the following terms and conditions:",
             "Do you agree to the above terms and conditions? (y/n): ",
         ])
+        choice = cls._choice_with_retries(t_and_c, ["y", "n"])
+        return choice.lower() == "y"
```

Comment:
I like the choice with retries a lot, you do not need the `lower()` here anymore though